README.md

```diff
@@ -1 +1,89 @@
-# powerai-image-memorability
\ No newline at end of file
+# Predicting Image Memorability with NASNet in Keras on PowerAI
+
+This code pattern will enable you to build an application that predicts how "unique" or "memorable" images are. You'll do this through the Keras deep learning library, using the NASNet architecture. The dataset this neural network will be trained on is called "LaMem" (Large-scale Image Memorability), by MIT. In order to process the 45,000 training images and 10,000 testing images (224x224 RGB) efficiently, we'll be training the neural network on a PowerAI machine on NIMBIX, enabling us to benefit from NVLink (direct CPU-GPU memory interconnect) without needing any extra code.
+
```

Comment:
Add a "what you're gonna learn section:

```
When the reader has completed this journey, they will understand how to:

Create a chatbot that converses via a web UI using Watson Assistant and Node.js
Use Watson Discovery with passage retrieval to find answers in FAQ documents
Use Watson Tone Analyzer to detect emotion in a conversation
Identify entities with Watson Natural Language Understanding
```

Also, you don't mention Kitura or Swift.

"To demonstrate this to the user, we'll be using Swift to do blah blah blah"

Reply:
Also mention CoreML